app/assets/javascripts/records.js

```diff
@@ -94,6 +94,28 @@
         return clean;
       }
 
+      var filtersExist = function() {
+        if ($('.filter-btn').next().length > 0) { return true; }
+        else { return false; }
+      }
```

Comment:
This should be `return $('.filter-btn').next().length > 0` there is never any need to do 
```
if(boolean) {return true}
else {return false}
```
It can always be replaced with `return boolean`